$NetBSD: patch-aa,v 1.4 1999/06/26 23:04:51 agc Exp $

Don't try to strip a script.
Remember to create the Perl example dir.

--- makefile.in	1998/10/05 12:07:06	1.1
+++ makefile.in	1998/10/05 12:07:54
@@ -51,7 +51,7 @@
 
 PERL		= @PERL@
 PERLLIB		= @perl_lib_path@
-INSTALL_SCRIPT	= $(srcdir)/sinstall.sh $(PERL) $(INSTALL_PROGRAM)
+INSTALL_SCRIPT	= @INSTALL@ -c -m 0755
 
 
 prefix		= @prefix@
@@ -615,6 +615,9 @@
 
 $(INSTALL_DIRS):
 	$(SHELL) ${srcdir}/mkdirs.sh $@
+
+${INSTALL_PERL_DIRS}:
+	${SHELL} ${srcdir}/mkdirs.sh $@
 
 ALWAYS:
 
